.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Setup dotnet
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: '3.1'
+          dotnet-version: '5.0'
```

Comment:
won't work, need to point on a prerelease.

you most likely want to use side by side loading.

```
      - name: Setup dotnet
        uses: actions/setup-dotnet@v1
        with:
          dotnet-version: '3.1'
      - name: Setup dotnet
        uses: actions/setup-dotnet@v1
        with:
          dotnet-version: '5.0.0-rc.2'
```

Reply:
Hey sorry, dot get the notification about this message :/
we ran here into one problem i think, the there is no [5.0.0-rc.2](https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.0-rc.2/dotnet-sdk-5.0.100-rc.2-linux-x64.tar.gz) repo but an [5.0.100-rc.2.20479.15](https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-rc.2.20479.15/dotnet-sdk-5.0.100-rc.2.20479.15-linux-x64.tar.gz) repo,

but seems the installer dont get the rc.2.2... repo or iam wrong?

Reply:
![index](https://user-images.githubusercontent.com/10728025/97310027-12bae800-1863-11eb-86c8-6e3734774e6c.png)
5.0.100-rc.2.20479.15 seem's for me the right sdk naming

Reply:
seems there some issues with the AzureCDN
https://github.com/dotnet/runtime/issues/34015